### PR TITLE
Like dislike functionality

### PR DESCRIPTION
**Pull request by varnita619**
added like dislike functionality
- add card to liked videos
- remove card from liked videos

Screenshot (36)

### CODE REVIEW

File: src/Reducer/LikedVideosReducer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                likes: action.payload,
+            }
+        case "ADD_TO_LIKES":
+            return {
+                ...state,
+                likes: action.payload,
+            };
+        case "REMOVE_FROM_LIKES":
+            return {
+                ...state,
```
Comment (pritam-kr) [inline src/Reducer/LikedVideosReducer.js:15]: Put all actions in separate file

**Review by pritam-kr — APPROVED** [review]
All looks good to me.

File: src/Components/AllRoutes/AllRoutes.jsx [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 import { HomePage } from "../../pages/Home/HomePage";
 import { VideoListing } from "../../pages/VideoListing/VideoListing";
 import { NavBar } from "../Navigation/NavBar";
-import {Login} from "../../pages/Auth/Login";
-import {Logout} from "../../pages/Auth/Logout";
-import {Signup} from "../../pages/Auth/Signup";
+import { Login } from "../../pages/Auth/Login";
+import { Logout } from "../../pages/Auth/Logout";
+import { Signup } from "../../pages/Auth/Signup";
+import { LikedVideos } from "../../pages/LikedVideos/LikedVideos";
```
Comment (anuj-kumary) [inline src/Components/AllRoutes/AllRoutes.jsx:9]: You can use here barrel import

**Review by samyakshah3008 — APPROVED** [review]
Clean UI 
Clean Code
Good naming conventions 
Good folder structure, but still you can move some code to different file and import it 
You can also give some comments in code where necessary for better code review experience 

Rest is all good for approval ✅

**Review by anuj-kumary — APPROVED** [review]
Code is clean, well structured, and properly documented
UI looks good
PR is ready to merge

**Pull request merged**